src/main/resources/processes/IEDET/IEDET_PSU_OUTCOME.bpmn

```diff
@@ -154,6 +157,15 @@
       <bpmndi:BPMNShape id="Activity_1an6g5s_di" bpmnElement="Activity_ResetCategories">
         <dc:Bounds x="460" y="200" width="100" height="80" />
       </bpmndi:BPMNShape>
+      <bpmndi:BPMNShape id="Event_0iu7swi_di" bpmnElement="EndEvent_Outcome">
+        <dc:Bounds x="832" y="319" width="36" height="36" />
+      </bpmndi:BPMNShape>
+      <bpmndi:BPMNShape id="Activity_15bvtsg_di" bpmnElement="Activity_ReturnCase">
+        <dc:Bounds x="720" y="200" width="100" height="80" />
+      </bpmndi:BPMNShape>
+      <bpmndi:BPMNShape id="Activity_1f3oa3j_di" bpmnElement="Service_UpdateDeadline">
```

Comment:
This can also be moved to PSU_IEDET_COMPLAINT, just after the PSU activity so that it looks like this:
<img width="651" alt="image" src="https://user-images.githubusercontent.com/92527240/203021993-ef04deba-ae1c-45b1-9eb6-cc8af6b23966.png">
That way you will only need to change that file and no others